vertx-web/src/test/java/io/vertx/ext/web/WebTestBase.java

```diff
@@ -125,7 +125,7 @@ protected void testRequestBuffer(HttpClient client, HttpMethod method, int port,
         latch.countDown();
       } else {
         resp.bodyHandler(buff -> {
-          assertEquals(responseBodyBuffer, buff);
+          assertEquals(responseBodyBuffer, normalizeLineEndingsFor(buff));
```

Comment:
I can get testRequest to pass a boolean to testRequestBuffer. Unfortunately I have class and it will take a bit.